content/hardware/04.pro/shields/portenta-vision-shield/tutorials/getting-started-camera/content.md

```diff
@@ -47,13 +47,23 @@ To capture the frames you will need to use the functions contained in `camera.h`
 
 ```cpp
 #include "camera.h"
-#include "himax.h"
+//For the Vision Shield Rev.1
+#include "himax.h"  // API to read from the Himax camera found on the Portenta Vision Shield Rev.1
```

Comment:
in the rest of the examples version 1 library is commented, should we do it here too?